09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
 
 ```{r, eval=FALSE, message=FALSE}
 link2GI::linkGDAL()
-cmd = paste("ogrinfo -ro -so -al", 
-            system.file("shape/nc.shp", package = "sf"))
+cmd = paste("ogrinfo -ro -so -al", system.file("shape/nc.shp", package = "sf"))
 system(cmd)
-#>INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
-#>    using driver `ESRI Shapefile' successful.
-#>
-#>Layer name: nc
-#>Metadata:
-#> DBF_DATE_LAST_UPDATE=2016-10-26
-#>Geometry: Polygon
-#>Feature Count: 100
-#>Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
-#>Layer SRS WKT:
-#>GEOGCS["GCS_North_American_1927",
-#>    DATUM["North_American_Datum_1927",
-#>        SPHEROID["Clarke_1866",6378206.4,294.9786982]],
-#>    PRIMEM["Greenwich",0],
-#>    UNIT["Degree",0.017453292519943295],
-#>    AUTHORITY["EPSG","4267"]]
-#>AREA: Real (24.15)
-#>PERIMETER: Real (24.15)
-#>CNTY_: Real (24.15)
-#>CNTY_ID: Real (24.15)
-#>NAME: String (80.0)
-#>FIPS: String (80.0)
-#>FIPSNO: Real (24.15)
-#>CRESS_ID: Integer (9.0)
-#>BIR74: Real (24.15)
-#>SID74: Real (24.15)
-#>NWBIR74: Real (24.15)
-#>BIR79: Real (24.15)
-#>SID79: Real (24.15)
-#>NWBIR79: Real (24.15)
-```
-
-This is of course a very simple example, and we could have achieved the same using `rgdal::ogrInfo()`.
-Still, it shows how we can use GDAL via the command-line from within R independently of other packages, and nothing stops us from using more advanced and/or not already implemented GDAL utilities in R.
+#> INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
+#>     using driver `ESRI Shapefile' successful.
+#> 
+#> Layer name: nc
+#> Metadata:
+#>  DBF_DATE_LAST_UPDATE=2016-10-26
+#> Geometry: Polygon
+#> Feature Count: 100
+#> Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
+#> Layer SRS WKT:
+#> GEOGCS["GCS_North_American_1927",
+#>     DATUM["North_American_Datum_1927",
+#>         SPHEROID["Clarke_1866",6378206.4,294.9786982]],
+#>     PRIMEM["Greenwich",0],
+#>     UNIT["Degree",0.017453292519943295],
+#>     AUTHORITY["EPSG","4267"]]
+#> AREA: Real (24.15)
+#> ...
+```
+
+This example --- which returns the same result as `rgdal::ogrInfo()` --- may be simple, but it shows how to use GDAL via the system command-line, independently of other packages.
+The 'link' to GDAL provided by **link2gi** could be used as a foundation for more doing more advanced GDAL work from the R or system CLI.^[
 Note also that the **RSAGA** package uses the command line interface to use SAGA geoalgorithms from within R (see section \@ref(rsaga)). 
-TauDEM (http://hydrology.usu.edu/taudem/taudem5/index.html) and the Orfeo Toolbox (https://www.orfeo-toolbox.org/) are further examples for offering a command line interface. 
+]
+TauDEM (http://hydrology.usu.edu/taudem/taudem5/index.html) and the Orfeo Toolbox (https://www.orfeo-toolbox.org/) are other low level spatial data processing libraries offering a command line interface. 
```

Comment:
Why low level? Don't know if these software package also come with GUI...

Reply:
Fixed, removed that statement.